hilti/runtime/include/type-info.h

```diff
@@ -518,7 +518,17 @@ class Function : public detail::NotImplementedType {};
 class Interval : public detail::AtomicType<hilti::rt::Interval> {};
 
 /** Auxiliary type information for type ``__library_type`. */
-class Library : public detail::ValueLessType {};
+class Library : public detail::AtomicType<hilti::rt::TypeInfo*> {
+public:
+    /**
+     * Constructor.
+     *
+     * @param cxx_name C++-side name of the type
+     */
+    Library(std::string cxx_name) : cxx_name(std::move(cxx_name)) {}
+
+    const std::string cxx_name; /**< C++-side name of type */
```

Comment:
Let's make this member `private` and add an accessor.

Reply:
Ok, will do. We're a bit inconsistent how type-infos handle visibility, but after another look, accessor seems to be more typical.